docs/surveys/salary-survey/2022.rst

```diff
@@ -14,7 +14,2069 @@ Write the Docs Salary Survey 2021 Results
+
+Proportion of role related to documentation
+-------------------------------------------
+
+.. container:: question
+   
+   .. rubric:: What we asked
+
+   Documentation is:
+
+   - the whole of my official job description
+   - part of my official job description
+   - not officially part of my job description, but I am expected to perform documentation-related tasks
+   - not officially part of my job description, and I am not expected to perform documentation-related tasks, but I do anyway
+
+   Approximately what percentage of your day-to-day tasks are documentation-related?
+
+   - 0-25%
+   - 26-50%
+   - 51-75%
+   - 76-100%
+
+As Write the Docs is a documentation-focused community, it's not surprising that the majority of respondents (73.1% in 2022) reported that documentation makes up both their whole official job description, and most or all of their day-to-day tasks (90.3% reporting more than 51% of their daily workload). Given the cross-disciplinary nature of documentation work, it's also not surprising that a proportion of respondents also work outside of that definition, and that there are even a small number documenting "by steath": those that are not officially designated as documentation employees or not required to perform documentation-related tasks, but do anyway.
+
+.. table:: Portion of role officially documentation-related (2022)
+   :width: 80%
+   :name: tbl-2022-portion-of-role-officially-documentation-related
+
+   +--------------------------------------+-----+-------+
+   | Portion of role                      | No. | %     |
+   +======================================+=====+=======+
+   | Wholly documentation                 | 384 | 73.1% |
+   +--------------------------------------+-----+-------+
+   | Partly documentation                 | 126 | 24.0% |
+   +--------------------------------------+-----+-------+
+   | Not documentation, but it's expected | 12  | 2.3%  |
+   +--------------------------------------+-----+-------+
+   | Not documentation, and not expected  | 3   | 0.6%  |
+   +--------------------------------------+-----+-------+
+
+.. table:: Portion of role actually documentation-related (2022)
+   :width: 80%
+   :name: tbl-2022-portion-of-role-actually-documentation-related
+
+   +-----------------+-----+-------+
+   | Portion of role | No. | %     |
+   +=================+=====+=======+
+   | 76%-100%        | 293 | 55.8% |
+   +-----------------+-----+-------+
+   | 51%-75%         | 160 | 30.5% |
+   +-----------------+-----+-------+
+   | 26%-50%         | 47  | 9.0%  |
+   +-----------------+-----+-------+
+   | 0-25%           | 25  | 4.8%  |
+   +-----------------+-----+-------+
+
+Interestingly, of the respondents who indicated that documentation was the whole of their official job description, 31% reported spending less than three quarters of their daily work time on documentation, with 5% reporting less than half their time. 
```

Comment:
This is more expected than interesting (to me). I wonder how we can dig into this next year?

Reply:
it's almost a separate survey in itself :)